.github/workflows/build-mac.yml

```diff
@@ -87,7 +88,84 @@ jobs:
+          security import certificate.p12 -k build.keychain -T /usr/bin/codesign -P "$MAC_BUILD_CERTIFICATE_PASSWORD"
+          security set-key-partition-list -S apple-tool:,apple:,codesign: -s -k "$MAC_BUILD_KEYCHAIN_PASSWORD" build.keychain
+          
+          # Confirm certificate
+          security find-identity
+          security find-certificate -a -p | openssl x509 -subject -noout
+          echo MAC_BUILD_CERTIFICATE_NAME="$MAC_BUILD_CERTIFICATE_NAME"
+          
+          # Move python modules to Resources
+          for f in config doc examples images locale plugin scripts Tktable2.11 ;
+            do
+              mv build/Arelle.app/Contents/MacOS/$f build/Arelle.app/Contents/Resources/
+            done
+          mkdir build/Arelle.app/Contents/Resources/lib
+          for f in `ls build/Arelle.app/Contents/MacOS/lib | grep -v "Python\|encodings\|codecs\|collections\|importlib\|matplotlib\|zlib\|library.zip\|so$"`
+            do
+              mv build/Arelle.app/Contents/MacOS/lib/$f build/Arelle.app/Contents/Resources/lib/
+            done
+          
+          # Code signing (*.so, *.dylib, etc)
+          for i in $(find build/Arelle.app -type f -perm +111 -exec file "{}" \; | \
+            grep -v "(for architecture" | \
+            grep -v -E "^- Mach-O" | \
+            grep -E "Mach-O executable|Mach-O 64-bit executable|Mach-O 64-bit bundle|Mach-O 64-bit dynamically linked shared library" | \
+            awk -F":" '{print $1}' | \
+            uniq)
+          do
+            codesign --deep --force --verify --verbose --timestamp \
+            --options runtime \
+            --sign "$MAC_BUILD_CERTIFICATE_NAME" \
+            "$i"
+          done
+          
+          find build/Arelle.app -type f -name "*.dylib*" -exec \
+            codesign --deep --force --verify --verbose --timestamp \
+            --options runtime \
+            --sign "$MAC_BUILD_CERTIFICATE_NAME" \
+            {} \;
+          
+          # Create symlinks (some libraries can't be code signed in /MacOS but need to be found within it)
+          ln -s ../../Resources/lib/tcltk build/Arelle.app/Contents/MacOS/lib/tcltk
+          ln -s ../../Resources/lib/tkinter build/Arelle.app/Contents/MacOS/lib/tkinter
+          # Symlink plugins folder so /Resources/plugin is found when searching in default /MacOS/plugin location
+          ln -s ../Resources/plugin build/Arelle.app/Contents/MacOS/plugin
+          
+          # Executable code signing (slightly redundant, but ensures all necessary files are code signed)
+          codesign --options runtime --force --deep --timestamp -s "$MAC_BUILD_CERTIFICATE_NAME" build/Arelle.app/Contents/MacOS/arelleCmdLine
+          codesign --options runtime --force --deep --timestamp -s "$MAC_BUILD_CERTIFICATE_NAME" build/Arelle.app/Contents/MacOS/arelleGUI
+      - name: Notarize app
+        uses: aaroncameron-wk/xcode-notarize@7b185cbfd79cf1fc2d2eaa597541329e18a44ba0
```

Comment:
Should we pull this into the arelle repo?

Reply:
I created a follow-up issue, I don't think a separate action is necessary. Since this won't be used long-term I don't think it's necessary to pull into the Arelle org.

Reply:
I created a follow-up issue, I don't think a separate action is necessary. Since this won't be used long-term I don't think it's necessary to pull into the Arelle org.